plugin.rb

```diff
@@ -351,7 +425,13 @@ def self.notify(id)
       uniq_func = proc { |i| i[:channel] }
       sort_func = proc { |a, b| precedence[a] <=> precedence[b] }
 
-      items = get_store(post.topic.category_id) | get_store("*") | get_store(0)
+      rows = get_category_store(post.topic.category_id) | get_category_store("*") | get_category_store(0)
+      post.topic.tags.each do |tag|
+        rows = rows + get_tag_store(tag.name)
+      end
+
+      ids = rows.map { |r| r.value unless r.nil? }
```

Comment:
If `r` is `nil`, your ids array will include `nil` as well which will cause the follow statement to fail

`rows.map(&:value).compact` is probably what you want here.